jboss-kie-common/added/launch/jboss-kie-common.sh

```diff
@@ -8,21 +8,21 @@ query_ocp_api() {
     local resourcePath="${2}"
     # only execute the following lines if this container is running on OpenShift
     if [ -e /var/run/secrets/kubernetes.io/serviceaccount/token ]; then
-        local namespace=$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)
-        local token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
-        local response=$(curl -s -w "%{http_code}" --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
+        namespace=$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)
+        token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+        response=$(curl -s -w "%{http_code}" --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
             -H "Authorization: Bearer $token" \
             -H 'Accept: application/json' \
-            ${KUBERNETES_SERVICE_PROTOCOL:-https}://${KUBERNETES_SERVICE_HOST:-kubernetes.default.svc}:${KUBERNETES_SERVICE_PORT:-443}/${ocpApi}/v1/namespaces/${namespace}/${resourcePath})
-        echo ${response}
+            "${KUBERNETES_SERVICE_PROTOCOL:-https}"://"${KUBERNETES_SERVICE_HOST:-kubernetes.default.svc}":"${KUBERNETES_SERVICE_PORT:-443}"/"${ocpApi}"/v1/namespaces/"${namespace}"/"${resourcePath}")
+        echo "${response}"
     fi
 }
 
 # Queries the Route from the Kubernetes API
 # ${1} - route name.
 query_route() {
     local routeName=${1}
-    echo $(query_ocp_api "apis/route.openshift.io" "routes/${routeName}")
+    query_ocp_api "apis/route.openshift.io" "routes/${routeName}"
```

Comment:
I don't think we can change it this way, it will not print/get the function return.